scripts/dist.ts

```diff
@@ -230,6 +233,15 @@ function toDist(sourcePath: string): YAML.Document {
     }
   }
 
+  if (computeFromWasExplicitlySet) {
+    if (groups.size === 1) {
+      logger.error(
+        `${id}: uses compute_from overall but it does not differ from per-key statuses. Delete this override.`,
```

Comment:
```suggestion
        `${id}: uses compute_from which should not be used given the overall status does not differ from the per-key statuses. Remove the compute_from override.`,
```

Reply:
I'm adapting your suggestion a little bit. We use "Consider deleting…" in several other messages, so I want this to be consistent—if you want to change this to "Remove" I'd welcome a PR fixing all of them. Also, this is a _must not_ not a _should not_. 😄

```suggestion
        `${id}: uses compute_from which must not be used when the overall status does not differ from the per-key statuses. Delete this override.`,
```